pkg/controller/utils/datadog/metrics_forwarder.go

```diff
@@ -112,22 +130,40 @@ func (mf *metricsForwarder) start(wg *sync.WaitGroup) {
 
 	mf.logger.Info("Datadog metrics forwarder initilized successfully")
 
+	// Send CR detection event
+	crEvent := crDetected(mf.id)
```

Comment:
it means each time the operator restart we have this event.
is it necessary?

Reply:
I thought it would make sense to have a CR detection event since we have a CR deletion event

Reply:
but we have already a Create Event too?

Reply:
if we dont, maybe we can create on if we see: "dad.Status" is empty

Reply:
we don't have a CR creation event, I named it `detection` event because at that point in the metric forwarder we only know that a CR exists (caused the creation of the metric forwarder itself)